lib/JITIDL/JITTypes.h

```diff
@@ -479,6 +479,7 @@ typedef struct FunctionBodyDataIDL
     boolean isParamAndBodyScopeMerged;
     boolean hasFinally;
     boolean usesArgumentsObject;
+    boolean doScopeObjectCreation;
```

Comment:
I know this is annoying, but midl hates implicit padding (for some reason) and gives a really heavy perf penalty for it. we need another 1 byte of padding here, and then another 2 bytes, and then another 4 bytes on x64 to 8 byte align the pointers.